From 1a0b1216dc82649b3c1a2a4090e05a4e63dfee9f Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 13 Jun 2026 04:56:33 +0200
Subject: [PATCH] fix metatypes dir

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4c531fc..d7c70b6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -9,7 +9,7 @@ qt_extract_metatypes(KF6Runner OUTPUT_FILES METATYPES_FILE)
 # see https://qt-project.atlassian.net/browse/QTBUG-123052
 # might be replaced with proper Qt API once that exists
 get_filename_component(METATYPES_FILE_NAME ${METATYPES_FILE} NAME)
-target_sources(KF6Runner INTERFACE $<INSTALL_INTERFACE:${KDE_INSTALL_QTMETATYPESDIR}/${METATYPES_FILE_NAME}>)
+target_sources(KF6Runner INTERFACE $<INSTALL_INTERFACE:lib/metatypes/${METATYPES_FILE_NAME}>)
 
 set_target_properties(KF6Runner PROPERTIES
     VERSION     ${KRUNNER_VERSION}
@@ -118,7 +118,7 @@ ecm_qt_install_logging_categories(
 
 ecm_generate_qdoc(KF6Runner krunner.qdocconf)
 
-install(FILES ${METATYPES_FILE} DESTINATION ${KDE_INSTALL_QTMETATYPESDIR})
+install(FILES ${METATYPES_FILE} DESTINATION ${KDE_INSTALL_LIBDIR}/metatypes)
 
 install(FILES
    "data/org.kde.krunner1.xml"
-- 
2.54.0

