From db017324eee6d435f212160cb28f846e64bb6ca7 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 13 Jun 2026 04:19:11 +0200
Subject: [PATCH] fix metatypes dir

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b7e18ea..457d04e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -156,7 +156,7 @@ qt_extract_metatypes(KF6GuiAddons OUTPUT_FILES METATYPES_FILE)
 # see https://qt-project.atlassian.net/browse/QTBUG-123052
 # might be replaced with proper Qt API once that exists
 get_filename_component(METATYPES_FILE_NAME ${METATYPES_FILE} NAME)
-target_sources(KF6GuiAddons INTERFACE $<INSTALL_INTERFACE:${KDE_INSTALL_QTMETATYPESDIR}/${METATYPES_FILE_NAME}>)
+target_sources(KF6GuiAddons INTERFACE $<INSTALL_INTERFACE:lib/metatypes/${METATYPES_FILE_NAME}>)
 
 ecm_generate_headers(KGuiAddons_HEADERS
   HEADER_NAMES
@@ -233,7 +233,7 @@ ecm_qt_install_logging_categories(
     DESTINATION "${KDE_INSTALL_LOGGINGCATEGORIESDIR}"
 )
 
-install(FILES ${METATYPES_FILE} DESTINATION ${KDE_INSTALL_QTMETATYPESDIR})
+install(FILES ${METATYPES_FILE} DESTINATION ${KDE_INSTALL_LIBDIR}/metatypes)
 
 if(BUILD_TESTING)
     add_executable(pasteclient systemclipboard/tests/paste.cpp)
-- 
2.54.0

