From 8d0759451db7d5462e1797e4d3d4e2b14790a1e5 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 13 Nov 2025 11:26:14 +0100
Subject: [PATCH] CMakeLists.txt: dont install desktop file

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c6058f..880834f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,8 +53,4 @@ add_executable(hyprland-welcome ${SRCFILES})
 
 target_link_libraries(hyprland-welcome PkgConfig::deps)
 
-install(
-  FILES contrib/hyprland-welcome.desktop
-  DESTINATION "share/applications")
-
-install(TARGETS hyprland-welcome)
\ No newline at end of file
+install(TARGETS hyprland-welcome)
-- 
2.51.0

