From 6505ccb3c48df921d7545f71a1543f89a5519b1f Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 23 May 2026 17:38:20 +0200
Subject: [PATCH] assets buildpath

Avoid buildpath issue

Upstream-Status: Pending
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ce124e71f..2e19bfc09 100644
--- a/meson.build
+++ b/meson.build
@@ -1039,7 +1039,7 @@ _noctalia_test_dependencies = _noctalia_system_dependencies + [
 ]
 
 _noctalia_compile_definitions = [
-  '-DNOCTALIA_SOURCE_ASSETS_DIR="' + meson.project_source_root() / 'assets' + '"',
+  '-DNOCTALIA_SOURCE_ASSETS_DIR="' + 'assets' + '"',
   '-DNOCTALIA_INSTALL_PREFIX="' + get_option('prefix') + '"',
   '-DNOCTALIA_INSTALL_DATADIR="' + get_option('datadir') + '"',
   '-DNOCTALIA_VERSION="' + meson.project_version() + '"',
-- 
2.54.0

