From 628c615bd108ceb04d7bda8db7c57fbcd47a0e52 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 29 Sep 2025 18:04:31 +0200
Subject: [PATCH] CMakeLists.txt: fix linking with opengl

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0fa337..ac99de1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ target_include_directories(
   PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
 set_target_properties(hyprtoolkit PROPERTIES VERSION ${HYPRTOOLKIT_VERSION}
                                               SOVERSION 6)
-target_link_libraries(hyprtoolkit PUBLIC OpenGL::EGL OpenGL::OpenGL
+target_link_libraries(hyprtoolkit PUBLIC EGL OpenGL
                                          PkgConfig::deps)
 
 check_include_file("sys/inotify.h" HAS_INOTIFY)
@@ -202,13 +202,13 @@ if(BUILD_TESTING)
     PUBLIC "./include"
     PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
   target_link_libraries(
-    hyprtoolkit_tests PRIVATE hyprtoolkit GTest::gtest_main OpenGL::EGL
-                                     OpenGL::OpenGL PkgConfig::deps)
+    hyprtoolkit_tests PRIVATE hyprtoolkit GTest::gtest_main EGL
+                                     OpenGL PkgConfig::deps)
 
   gtest_discover_tests(hyprtoolkit_tests)
 
   add_executable(embeddedRendererTest "tests/EmbeddedRenderer.cpp")
-  target_link_libraries(embeddedRendererTest PRIVATE hyprtoolkit GTest::gtest_main OpenGL::EGL OpenGL::OpenGL PkgConfig::deps)
+  target_link_libraries(embeddedRendererTest PRIVATE hyprtoolkit GTest::gtest_main EGL OpenGL PkgConfig::deps)
   gtest_discover_tests(embeddedRendererTest)
 
   # Add coverage to hyprtoolkit
