From 628c615bd108ceb04d7bda8db7c57fbcd47a0e52 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 29 Sep 2025 18:04:31 +0200
Subject: [PATCH] CMakeLists.txt: fix linking with opengl

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b47f0b0..55e9176 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -77,7 +77,7 @@ target_include_directories(
   PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
 set_target_properties(hyprtoolkit PROPERTIES VERSION ${HYPRTOOLKIT_VERSION}
                                             SOVERSION 2)
-target_link_libraries(hyprtoolkit PUBLIC OpenGL::EGL OpenGL::OpenGL PkgConfig::deps)
+target_link_libraries(hyprtoolkit PUBLIC EGL GL PkgConfig::deps)
 
 check_include_file("sys/timerfd.h" HAS_TIMERFD)
 pkg_check_modules(epoll IMPORTED_TARGET epoll-shim)
@@ -111,7 +111,7 @@ if (NOT DISABLE_TESTS)
   target_compile_options(hyprtoolkit_inline_tests PRIVATE --coverage)
   target_link_options(hyprtoolkit_inline_tests PRIVATE --coverage)
   target_include_directories(hyprtoolkit_inline_tests PUBLIC "./include" PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
-  target_link_libraries(hyprtoolkit_inline_tests PRIVATE GTest::gtest_main OpenGL::EGL OpenGL::OpenGL PkgConfig::deps)
+  target_link_libraries(hyprtoolkit_inline_tests PRIVATE GTest::gtest_main EGL GL PkgConfig::deps)
   gtest_discover_tests(hyprtoolkit_inline_tests)
 endif()
 
-- 
2.51.0

