From 628c615bd108ceb04d7bda8db7c57fbcd47a0e52 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 29 Sep 2025 18:04:31 +0200
Subject: [PATCH] CMakeLists.txt: fix linking with opengl

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f16d7ce..74d587f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,7 +73,7 @@ target_include_directories(
   PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
 set_target_properties(hyprtoolkit PROPERTIES VERSION ${HYPRTOOLKIT_VERSION}
                                              SOVERSION 5)
-target_link_libraries(hyprtoolkit PUBLIC OpenGL::EGL OpenGL::OpenGL
+target_link_libraries(hyprtoolkit PUBLIC EGL
                                          PkgConfig::deps)
 
 check_include_file("sys/timerfd.h" HAS_TIMERFD)
@@ -171,8 +171,8 @@ if(BUILD_TESTING)
     PUBLIC "./include"
     PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
   target_link_libraries(
-    hyprtoolkit_tests PRIVATE hyprtoolkit GTest::gtest_main OpenGL::EGL
-                                     OpenGL::OpenGL PkgConfig::deps)
+    hyprtoolkit_tests PRIVATE hyprtoolkit GTest::gtest_main EGL
+                                     PkgConfig::deps)
 
   gtest_discover_tests(hyprtoolkit_tests)
 
-- 
2.51.0

