From 014ae5e2f9775213e231d4a02530e2d38a2ef5fc Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 13 Jun 2025 19:22:18 +0200
Subject: [PATCH] CMakeLists.txt: fix linking with opengl

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 4 ++--
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 475c744..e5426dd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,7 +67,7 @@ target_include_directories(
   PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
 set_target_properties(aquamarine PROPERTIES VERSION ${AQUAMARINE_VERSION}
                                             SOVERSION 9)
-target_link_libraries(aquamarine OpenGL::EGL OpenGL::OpenGL PkgConfig::deps)
+target_link_libraries(aquamarine EGL GL PkgConfig::deps)
 
 check_include_file("sys/timerfd.h" HAS_TIMERFD)
 pkg_check_modules(epoll IMPORTED_TARGET epoll-shim)

-- 
2.49.0

