From 15a51707cd18b398bf3a3abc077fd0f8245b3ceb Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 30 Dec 2024 08:26:09 +0100
Subject: [PATCH] hyprpicker.cpp: drop wl_ressource

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/hyprpicker.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/hyprpicker.cpp b/src/hyprpicker.cpp
index 2e8dbbf..ed07b99 100644
--- a/src/hyprpicker.cpp
+++ b/src/hyprpicker.cpp
@@ -573,7 +573,7 @@ void CHyprpicker::initKeyboard() {
 }
 
 void CHyprpicker::initMouse() {
-    m_pPointer->setEnter([this](CCWlPointer* r, uint32_t serial, wl_resource* surface, wl_fixed_t surface_x, wl_fixed_t surface_y) {
+    m_pPointer->setEnter([this](CCWlPointer* r, uint32_t serial, wl_proxy* surface, wl_fixed_t surface_x, wl_fixed_t surface_y) {
         auto x = wl_fixed_to_double(surface_x);
         auto y = wl_fixed_to_double(surface_y);
 
@@ -590,7 +590,7 @@ void CHyprpicker::initMouse() {
 
         m_pCursorShapeDevice->sendSetShape(serial, WP_CURSOR_SHAPE_DEVICE_V1_SHAPE_CROSSHAIR);
     });
-    m_pPointer->setLeave([this](CCWlPointer* r, uint32_t timeMs, wl_resource* surf) {
+    m_pPointer->setLeave([this](CCWlPointer* r, uint32_t timeMs, wl_proxy* surf) {
         for (auto& ls : m_vLayerSurfaces) {
             if (ls->pSurface->resource() == surf) {
                 renderSurface(ls.get(), true);
-- 
2.47.1

