From 57f01e16279d2ae0c4e05e6c2b3bb4fa0ab5a5be Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Wed, 20 May 2026 18:55:49 +0200
Subject: [PATCH] CMakeLists.txt: dont be specific about required lua
 version

Upstream-Status: Pending
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c9116a6..3de4bec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,7 @@ pkg_check_modules(deps REQUIRED IMPORTED_TARGET
     pixman-1
     wayland-server
     xkbcommon
-    lua5.4
+    lua
 )
 target_link_libraries(hyprexpo PRIVATE rt PkgConfig::deps)
 
-- 
2.54.0

