From 42d3806947e68ffe864a0c37210310c1e3457a8c Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 13 Jun 2025 19:04:22 +0200
Subject: [PATCH] CMakeLists.txt: use find_program for OpenGL

Signed-off-by: Markus Volk <f_l_k@t-online.de>

Upstream-Status: Pending
---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4738a81..ffe2a86 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,10 +122,11 @@ include(CPack)
 
 target_link_libraries(hyprpaper PkgConfig::deps)
 
+find_package(OpenGL REQUIRED COMPONENTS EGL GLES2)
+
 target_link_libraries(
   hyprpaper
-  OpenGL
-  GLESv2
+  OpenGL::EGL OpenGL::GLES2
   pthread
   magic
   ${CMAKE_THREAD_LIBS_INIT}
-- 
2.49.0

